dependencies.props

```diff
@@ -1,4 +1,4 @@
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+a1e3835a389e9953fba5a96988bf8b670fcae207<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I'll go ahead and try to put together the fix now, since I'm familiar with this problem.